### DISCUSSION TITLE
How can I see which user access the wireguard VPN?

### DISCUSSION THREAD

**Discussion by genieai-vikas** [wg-easy/wg-easy#526] [Q&A]
How can I monitor the connections of all users to the WireGuard VPN? My VPN is functioning properly, but I am unsure where to check for the access logs

**Comment by DigneZzZ** [comment]
You can see it in your WG-easy web-gui

**Comment by jpergentino** [comment]
That's a valid question. Is there a way to access the history of connections and disconnections?

Thanks in advance!

**Comment by eandersons** [comment]
WireGuard does not have access logs/connection history, but there are third-party tools that provide such functionality (see https://github.com/wg-easy/wg-easy/discussions/732#discussioncomment-8029651).

**Comment by latimeria68** [comment]
Hello,
any way at least to see the real IP address of the connected clients inside the web-gui? 
Thanks
